src/assets/create.js

```diff
@@ -1,36 +1,28 @@
-class UI {
-  static displayTasks = () => {
-    const toDoList = [
-      {
-        description: 'Wash car',
-        index: 1,
-        completed: true,
-      },
-      {
-        description: 'Take dog for walk',
-        index: 2,
-        completed: false,
-      },
-    ];
+import Storage from './store.js';
 
-    const toDo = toDoList;
+class UI {
+  static displayTasks() {
     const container = document.querySelector('#list-items');
     container.innerHTML = '';
-    toDo.forEach((list) => {
+    const listItems = Storage.getList();
+    listItems.forEach((task) => {
       const listCont = document.createElement('div');
       listCont.className = 'single-task';
-      const label = document.createElement('label');
-      label.setAttribute('for', 'id1');
-      label.innerHTML = list.description;
-      const toDo = document.createElement('input');
-      toDo.setAttribute('type', 'checkbox');
-      toDo.id = 'id1';
-      const dots = document.createElement('i');
-      dots.className = 'bi bi-three-dots-vertical';
-      listCont.append(toDo, label, dots);
+      listCont.setAttribute('data-index', task.index);
+      listCont.innerHTML = `
+      <label for"id1"></label>
+      <input class="input" type="checkbox" id="id1">
+         <span class="span" contenteditable="true">${task.description}</span>
```

Comment:
- [ ] I can edit the to-do item but it is not changing in the local storage. Kindly fix it.
![image](https://user-images.githubusercontent.com/38635158/184502537-73a089ea-f2b8-4120-ad94-baeb99920f60.png)